roles/common/tasks/config_validations.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Retrieve SSL public key hash from private key on Local Host
   shell:
-    cmd: openssl pkey -pubout | openssl md5
+    cmd: openssl pkey -pubout | openssl sha256
```

Comment:
We should use openssl 3

Reply:
yes, I'm working on adding openssl 3. By default, rhel 9 comes with openssl 3.0 but rhel 8 comes with openssl 1.1, I will  add tasks to use openssl 3.0 in rhel 8

Reply:
update: Rhel 8 doesn't have a yum/dnf package for installing openssl3. So we need to install openssl3 from [source](https://maggiminutes.com/install-latest-openssl-3-on-linux/) in rhel 8.  After installation, we need to enable fips manually by [editing](https://www.openssl.org/docs/man3.0/man7/fips_module.html#:~:text=Edit%20the%20config%20file%20to%20add%20the%20following%20lines%20near%20the%20beginning) the openssl.cnf file. Due to this, we decided to go ahead with openssl1.1 in RHEL 8.